db/migrate/20230925222043_create_tasks.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTasks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :tasks do |t|
+      t.string :name
+      t.text :description
+      t.string :status
```

Comment:
You need to add constraints in db level too

```suggestion
      t.string :name, null: false
      t.text :description
      t.string :status, null: false
```